src/components/sendV2/transactionStatus/transactionStatus.js

```diff
@@ -58,7 +52,18 @@ class TransactionStatus extends React.Component {
           <p>{this.props.t('{{paragraph}}', { paragraph: transactionStatus.bodyText.paragraph })}</p>
```

Comment:
This won't actually work, this just interpolates the value, not translating it, the `paragraph` should be translated before.

Reply:
done